jina/serve/streamer.py

```diff
@@ -221,3 +221,34 @@ def get_streamer():
     @staticmethod
     def _set_env_streamer_args(**kwargs):
         os.environ['JINA_STREAMER_ARGS'] = json.dumps(kwargs)
+
+
+class ExecutorStreamer:
```

Comment:
This is not a good name, unless the `request_size` is something you consider

Reply:
Should we batch the requests then? Or keep this one simple and not have `request_size`?

Reply:
this we should discuss now

Reply:
decided to respect request_size